autodevops_simple_app.yml

```diff
@@ -119,7 +108,7 @@ Deploy app (dev):
   variables:
     HOST: ${CI_ENVIRONMENT_SLUG}-${CI_PROJECT_NAME}.${KUBE_INGRESS_BASE_DOMAIN}
   environment:
-    name: review/${CI_COMMIT_REF_NAME}-dev
+    name: ${CI_COMMIT_REF_NAME}-dev
```

Comment:
would be good to keep review prefix to keep folded environments ?

Reply:
Alright
i just that one can not see pods details on in the folders
![image](https://user-images.githubusercontent.com/730511/78264949-191ef680-7504-11ea-932d-e6fe366458ba.png)

Reply:
i think it works in gitlab.com ?

Reply:
> i think it works in gitlab.com ?

dunno

Reply:
en effet je trouve pas de preuve que ca marche :/